fa20/syllabus.yml

```diff
@@ -21,12 +21,12 @@
     authors: []
     filename: meeting01
```

Comment:
Filename can't be default. 😕  Since it's "video games" why not "ai-video-games" or something similar?